packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -354,6 +373,18 @@ mod tests {
             "difference calculated by complement",
         );
 
+        if lhs.merge(rhs) == lhs {
+            assert!(
+                lhs.contains_interval(&rhs),
+                "{lhs:?} contains {rhs:?}, but does not report so"
```

Comment:
I don't quite get this, what do you mean by "but does not report so"

Reply:
`contains_interval` should return true, so that `lhs` contains `rhs`, but it reports `false`

Reply:
Then perhaps we should say 
```suggestion
                "{lhs:?} contains {rhs:?}, but `contains_interval` reported otherwise"
```

Reply:
I also changed this for the other messages in 8b83780096846b94935db84c9f9172266b46d685